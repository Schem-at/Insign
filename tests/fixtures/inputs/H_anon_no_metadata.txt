@rc([0,0,0],[3,3,3])

@rc([10,0,0],[13,3,3])

@named=rc([20,0,0],[23,3,3])
#named:has_metadata=true

@rc([30,0,0],[33,3,3])
#doc.label="This anon has metadata"
