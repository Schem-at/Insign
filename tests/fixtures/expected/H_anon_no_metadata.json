{
  "__anon_0_4": {
    "bounding_boxes": [
      [
        [
          30,
          0,
          0
        ],
        [
          33,
          3,
          3
        ]
      ]
    ],
    "metadata": {
      "doc.label": "This anon has metadata"
    }
  },
  "named": {
    "bounding_boxes": [
      [
        [
          20,
          0,
          0
        ],
        [
          23,
          3,
          3
        ]
      ]
    ],
    "metadata": {
      "has_metadata": true
    }
  }
}